gate-plugins/src/main/kotlin/com/netflix/spinnaker/gate/plugins/deck/DeckPluginsController.kt

```diff
@@ -31,7 +30,6 @@ import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @RequestMapping("/plugins/deck")
-@ConditionalOnBean(DeckPluginService::class)
 class DeckPluginsController(
   private val deckPluginService: DeckPluginService
```

Comment:
If we're going this route, we should probably wrap `DeckPluginService` in `Provider` / `ObjectProvider`.

Reply:
Why? In going this route, the bean will always be available.